static/app/views/alerts/rules/uptime/uptimeChecksGrid.tsx

```diff
@@ -182,8 +188,10 @@ function CheckInBodyCell({
 
       return (
         <TraceCell>
-          {spanCount ? (
-            <Link to={`/performance/trace/${traceId}/`}>{getShortEventId(traceId)}</Link>
+          {alwaysShowTraceLink || spanCount ? (
+            <Link to={`/performance/trace/${traceId}/?includeUptime=1`}>
+              {getShortEventId(traceId)}
```

Comment:
Note: We should use `getTraceDetailsUrl` here eventually, to have the correct breadcrumbs in the trace view header, add a new source 'uptime'. The function is a bit cloudy atm lmk if you need help. I'll it up over quality quarter.